### DISCUSSION TITLE
not been able to read the mac address of Scanned devices

### DISCUSSION THREAD

**Discussion by siki411** [shinyorg/shiny#1378] [Q&A] [answered]
Hi there!
i am having this problem while using shiny ble for scanning ble devices . Actually i am not been able to read the device mac address.. apart from that i am able to read the device name device.Peripheral.Name, but not been able to read the device mac address.. i am attaching the snippet of the code for better idea ... can you please tell me any function with which i can read the mac address of the device.. or there is no such function for this task .. though i tried it with uuid assuming that it might had mac address of the device but i was wrong .. i hope you could guide me better.
thanks 
![question](https://github.com/shinyorg/shiny/assets/61039629/d4acdb69-f774-4656-a490-d5b0ea6f3a39)

**Comment by aritchie** [comment]
You cannot read the MAC address on Apple platforms

**Reply by siki411** [reply]
ohh so because of that this is not included in library. Because it will create cross-compatibility issue between android and ios ?

**Reply by aritchie** [reply, marked as answer]
Correct.  I believe Android is working to deprecate the MAC address in the future anyhow
